content/en/docs/components/pipelines/v2/cli.md

```diff
@@ -4,119 +4,161 @@ description = "Interact with KFP via the CLI"
 weight = 8
 +++
 
+<!-- TODO: Improve or standardize rendering of variables and placeholders -->
+<!-- TODO: Standardize inline references to KFP CLI SDK -->
+
 <!-- TODO: use /latest instead of /master when SDK goes GA -->
-This section provides a high-level summary of the KFP command line interface (CLI). For more detailed API documentation, see the [KFP CLI reference][cli-reference-docs].
+This section provides a summary of the available commands in the KFP CLI. For more comprehensive documentation about all the available commands in the KFP CLI, see [Command Line Interface][cli-reference-docs] in the [KFP SDK API reference][kfp-sdk-api-ref].
 
 ## Usage
 The KFP CLI is installed with the KFP SDK as `kfp`.
 
-You can check that the KFP CLI is installed in your environment by running:
+### Check availability of KFP CLI
+
+To check whether KFP CLI is installed in your environment, run the following command:
 
 ```shell
 kfp --version
 ```
 
-All commands assume the general structure:
+### General syntax
+
+All commands in the KFP CLI use the following general syntax:
 
 ```shell
 kfp [OPTIONS] COMMAND [ARGS]...
 ```
 
-For example, to list all runs for a specific endpoint:
+For example, to list all runs for a specific endpoint, run the following command:
 
 ```shell
 kfp --endpoint http://my_kfp_endpoint.com run list
 ```
 
-You can get help for a particular command directly on the command line using `--help`:
+### Get help for a command
+
+To get help for a specific command, use the argument `--help` directly in the command line. For example, to view guidance about the `kfp run` command, type the following command:
 
 ```shell
 kfp run --help
 ```
+## Main functons of the KFP CLI
+
+You can use the KFP CLI to do the following:
+
+* [Interact with KFP resources](#cli_interact_with_kfp_resources)
+
+* [Compile pipelines](#cli_compile_pipelines)
 
-The KFP CLI serves three main functions:
-1. Interacting with KFP resources
-2. Compiling pipelines
-3. Building Python-based containerized components
+* [Build containerized Python components](#cli_build_python_components)
 
-## Interact with KFP resources
-The majority of the KFP CLI commands involve creating, reading, updating, or deleting KFP resources via the KFP backend. All of these commands take a common structure:
+### Interact with KFP resources<a id="cli_interact_with_kfp_resources"></a>
```

Comment:
When I click on the links above, it's not jumping to the block of text I expect.

Can you try using the markdown-native header reference syntax and see if it resolves this?

```markdown
## Multitoken header 

[some linked text](#multitoken-header)
```

Reply:
The markdown native headers didn't work for some reason. When I check the preview, the links seem to work fine.